packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -441,6 +450,7 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
         if (msg.sender != guardian()) revert Unauthorized();
         respectedGameType = _gameType;
         respectedGameTypeUpdatedAt = uint64(block.timestamp);
+        emit RespectedGameTypeSet(_gameType);
```

Comment:
`eth_getLogs` doesn't return timestamps when you query logs (so a second RPC call to `eth_getBlock` is needed if you want timestamp, though I think reth did improve this for `eth_getLogs`), and since we save the timestamp to storage it might be useful to also want to include in the log?

Reply:
This event is shared with the signature of `DeputyGuardianModule`. We can change it, though I believe that code is currently in audit.

Reply:
> This event is shared with the signature of the event in `DeputyGuardianModule`

Ah good point. @maurelian any thoughts here? I'd be ok with having them differ (so only this one has timestamp), or with changing both to have timestamp (it's a small change, so it's ok IMO)

Reply:
yeah, I think making a tiny to a DGM event is fine.